apikit/convenience.py

```diff
@@ -287,6 +282,11 @@ def __init__(self, reason, status_code=None, content=None):
                 raise TypeError("'content' must be a basestring")
         self.content = content
 
+    def __str__(self):
+        """Useful textual representation"""
+        return "BackendError: %d %s [%s]" % (self.status_code,
+                                             self.reason, self.content)
```

Comment:
SHAME! But I don't think we have a strong opinion on % versus using format method when manipulating strings.